tests/snapshot.rs

```diff
@@ -218,7 +218,7 @@ fn get_audios() -> Vec<PathBuf> {
     for entry in fs::read_dir(&audio_dir).unwrap() {
         let entry = entry.unwrap();
         let path = entry.path();
-        if !path.is_dir() {
```

Comment:
Do you still need this change now the mp3 is gone?

Reply:
I'm planning to add it back in a follow up PR where I'll add audio file reading code so let's just keep it there.